source/tools/BuildTool.rst

```diff
@@ -0,0 +1,51 @@
+.. _build-tool:
+
+Build Tool
+-----------
+
+``BuildLoader.py`` compiles source code with all the *magics* to generate output image(s). It provides two subcommands: build and clean.
+
+
+You can build |SPN| with the following options:
+
+* Build for a supported target
+* Debug or release build
+* Use debug image of FSP
+* Change your payload files
+* Attach a version data structure of your own
+
+Command Syntax::
+
+    python BuildLoader.py <subcommand> <target> <options>
+
+    <subcommand>  : build or clean
+    <target>      : board name (e.g. apl or qemu)
+
+
+Windows and Linux share the same command line options as follows. For example, for ``build`` subcommand::
+
+    usage: BuildLoader.py build [-h] [-r] [-v] [-fp FSPPATH] [-fd] [-p PAYLOAD] board
+
+    positional arguments:
+    board                 Board Name (apl, qemu)
+
+    optional arguments:
+      -h, --help                      Show this help message and exit
+      -r, --release                   Release build
+      -v, --usever                    Use board version file
+      -fp FSPPATH                     FSP binary path relative to FspBin in Silicon folder
+      -fd, --fspdebug                 Use debug FSP binary
+      -p PAYLOAD, --payload PAYLOAD   Payload file name
+
+
+For a list of platforms supported::
+
+  pthon BuildLoader.py build -h
```

Comment:
typo for "pthon".   Also this does not work, use below instead
python  BuildLoader.py build ?